### PR TITLE
Add support for Croatia

### LINKED ISSUE #5972: Add support for Croatia
Add support for Croatia as part of the new international expansion project.

The following is a checklist to help with this:
- [ ] Add the country to `supported_countries` function
- [ ] Add the country to `countryFeeStripeDocsSectionNumbers` function
- [ ] Onboard on Stripe using a test account form the country (make sure to share it in PR for testers)
- [ ] Make a payment using a test card
- [ ] Make sure the currency is showing properly
- [ ] Turn on multiple payment methods and make a test payment with one
- [ ] Simulate a dispute using one of the [test cards](https://stripe.com/docs/testing?testing-method=card-numbers#disputes). Make sure the disputes are working properly (ie, showing in the list + currency is good)
- [ ] Make sure the deposits are working properly (ie, double check with Stripe Docs the minimum deposits amount, frequency, wait time...)

### PR DESCRIPTION
**Pull request by dpaun1985**
Fixes #5972 

#### Changes proposed in this Pull Request

Add support for Croatia

#### Testing instructions
- On server side checkout 3302-gh-Automattic/woocommerce-payments-server branch
- **Base fees**
- Go to Woo -> Settings -> General and set Store Country to Croatia and currency to Euro
- Remove supported countries from the account on server side
- Place an order - base fees should be applied
- **FX fees**
- change store currency to USD
- Remove supported countries from the account on server side
- Place an order - base and FX fees should be applied
- **EPS fees**
- Place an order with EPS
- You will be redirected to a Stripe test page where you can approve/fail the order
- Approve the order - EPS fees should be applied

Continue with all fees defined for Croatia.

-------------------

- [ ] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `6069` or branch name `add/croatia-fees` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&branches.woocommerce-payments=add/croatia-fees) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: 4a7a6c47fa60e0b3f16d829db604ec9d7430dbb7
- Build time: 2023-05-04 09:17:12 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: changelog/add-croatia-fees [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: major
+Type: add
+
+Add support for Croatia fees
```
Comment (anu-rock) [inline changelog/add-croatia-fees]: I think we can exclude the word "fees" here.

**Review by anu-rock — APPROVED** [review]
LGTM (code-only review). Pre-approving with a minor comment.

File: changelog/add-croatia-fees (hunk 1 above)
Comment (dpaun1985) [reply changelog/add-croatia-fees]: changed. thx

**Comment by elazzabi** [comment]
Removed the changelog in https://github.com/Automattic/woocommerce-payments/pull/6069/commits/4a7a6c47fa60e0b3f16d829db604ec9d7430dbb7 given we'll have one changelog for the 6 countries

**Pull request merged**